integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -431,8 +473,22 @@ private void checkTableReadWrite(SparkTableInfo table) {
             .map(Object::toString)
             .map(
                 s -> {
-                  String tmp = org.apache.commons.lang3.StringUtils.removeEnd(s, "'");
-                  tmp = org.apache.commons.lang3.StringUtils.removeStart(tmp, "'");
+                  String tmp = org.apache.commons.lang3.StringUtils.remove(s, "'");
+                  if (org.apache.commons.lang3.StringUtils.isEmpty(tmp)) {
+                    return tmp;
+                  } else if (tmp.startsWith("array")) {
```

Comment:
could you add some description about why you do this?  better to provide some example

Reply:
updated.